src/models/comments_model.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose, { Document, Schema, ObjectId } from "mongoose";
+
+export interface ICommentDocument extends Document {
+  _id: ObjectId,
```

Comment:
```suggestion
  _id: ObjectId;
```
Annoying but should be fixed 😬